packages/renderer/src/lib/pod/DeployPodToKube.spec.ts

```diff
@@ -165,7 +164,7 @@ beforeEach(() => {
   listSimpleContainersByLabelMock.mockResolvedValue([simpleContainerInfo]);
 });
 
-async function waitRender(customProperties: any): Promise<void> {
+async function waitRender(customProperties: object): Promise<void> {
```

Comment:
object is no better than any here.

```suggestion
async function waitRender(customProperties:  Partial<ComponentProps<DeployPodToKube>>): Promise<void> {
```

The waitRender here take some properties that could be passed to the render function. So the render function takes `ComponentProps<T>`; but if we use this type, we will have to provide all  the props everytime we call it, so we can use Partial to make all props optional.